apps/bilateral_grid/Makefile

```diff
@@ -1,22 +1,26 @@
 include ../support/Makefile.inc
 
-all: filter
+all: $(BIN)/filter
 
-bilateral_grid: bilateral_grid.cpp
-	$(CXX) $(CXXFLAGS) bilateral_grid.cpp -g $(LIB_HALIDE) -o bilateral_grid -lz $(LDFLAGS) \
-	$(LLVM_SHARED_LIBS)
+$(BIN)/bilateral_grid_exec: bilateral_grid_generator.cpp $(GENERATOR_DEPS)
+	@-mkdir -p $(BIN)
+	$(CXX) $(CXXFLAGS) $(LDFLAGS) -fno-rtti $(filter-out %.h,$^) -o $@
```

Comment:
I think LDFLAGS might need to be at the end of the line. There's a failure where we're missing symbols that should be in ole32

https://buildbot.halide-lang.org/master/builders/mingw64-64-trunk/builds/1558/steps/make%20test_apps/logs/stdio

Perhaps it's because the -lole32 comes before the libHalide.a?